### ISSUE TITLE
ARM CI failing on multiple CI jobs from tests/runtime repo

### ISSUE THREAD

**Issue by likebreath** [kata-containers/packaging#1118]
# Description of problem

Multiple ARM CI jobs from tests/runtime repo are now failing for the same error while building the kata kernel. Failing CI jobs include:

`kata-containers-runtime-ARM-18.04-PR`: latest failing log [here](http://jenkins.katacontainers.io/job/kata-containers-runtime-ARM-18.04-PR/1738/console)
`kata-containers-tests-ARM-18.04-PR-firecracke`: latest failing log [here](http://jenkins.katacontainers.io/job/kata-containers-tests-ARM-18.04-PR-firecracker/169/console)
`kata-containers-runtime-ARM-18.04-PR-firecracke`: latest failing log [here](http://jenkins.katacontainers.io/job/kata-containers-runtime-ARM-18.04-PR-firecracker/103/console)

# Expected result

The CI jobs mentioned above should be passing.

# Actual result

All CI jobs are failing for the following error:
```
09:24:30 linux-5.4.60.tar.xz: OK
09:24:51 INFO: Applying patch /home/jenkins/workspace/kata-containers-tests-ARM-18.04-PR-firecracker/go/src/github.com/kata-containers/packaging/kernel/patches//5.4.x/0001-NO-UPSTREAM-9P-always-use-cached-inode-to-fill-in-v9.patch
09:24:51 patching file fs/9p/vfs_inode.c
09:24:51 Hunk #1 succeeded at 1059 (offset -21 lines).
09:24:51 patching file fs/9p/vfs_inode_dotl.c
09:24:51 Hunk #1 succeeded at 465 (offset -15 lines).
09:24:51 INFO: Applying patch /home/jenkins/workspace/kata-containers-tests-ARM-18.04-PR-firecracker/go/src/github.com/kata-containers/packaging/kernel/patches//5.4.x/0002-net-virtio_vsock-Fix-race-condition-between-bind-and.patch
09:24:51 patching file net/vmw_vsock/virtio_transport_common.c
09:24:51 INFO: Applying patch /home/jenkins/workspace/kata-containers-tests-ARM-18.04-PR-firecracker/go/src/github.com/kata-containers/packaging/kernel/patches//5.4.x/0003-arm-arm64-Provide-a-wrapper-for-SMCCC-1.1-calls.patch
09:24:51 patching file include/linux/arm-smccc.h
09:24:51 INFO: Applying patch /home/jenkins/workspace/kata-containers-tests-ARM-18.04-PR-firecracker/go/src/github.com/kata-containers/packaging/kernel/patches//5.4.x/0004-arm-arm64-smccc-psci-add-arm_smccc_1_1_get_conduit.patch
09:24:51 patching file drivers/firmware/psci/psci.c
09:24:51 patching file include/linux/arm-smccc.h
09:24:51 ls: cannot access '/home/jenkins/workspace/kata-containers-tests-ARM-18.04-PR-firecracker/go/src/github.com/kata-containers/packaging/kernel/configs/fragments/arm64/../common/experimental/*.conf': No such file or directory
09:24:51 INFO: Constructing config from fragments: /home/jenkins/workspace/kata-containers-tests-ARM-18.04-PR-firecracker/go/src/github.com/kata-containers/packaging/kernel/configs/fragments/arm64/.config
09:25:09 INFO: Some CONFIG elements failed to make the final .config:
09:25:09 INFO: Value requested for CONFIG_MEMORY_HOTREMOVE not in final .config
09:25:09 Value requested for CONFIG_ZONE_DEVICE not in final .config
09:25:09 Value requested for CONFIG_DEV_PAGEMAP_OPS not in final .config
09:25:09 Value requested for CONFIG_ND_PFN not in final .config
09:25:09 Value requested for CONFIG_NVDIMM_PFN not in final .config
09:25:09 Value requested for CONFIG_NVDIMM_DAX not in final .config
09:25:09 INFO: Generated config file can be found in /home/jenkins/workspace/kata-containers-tests-ARM-18.04-PR-firecracker/go/src/github.com/kata-containers/packaging/kernel/configs/fragments/arm64/.config
09:25:09 ERROR: Failed to construct requested .config file
09:25:09 ERROR: failed to find default config 
09:25:09 Failed at 80: "${kernel_repo_dir}/kernel/build-kernel.sh" -v "${kernel_version}" "setup"
09:25:15 Failed at 31: "${cidir}/install_kata_kernel.sh" "${tag}"
09:25:15 Build step 'Execute shell' marked build as failure
```

**Comment by jongwu** [comment]
thanks @likebreath , I should have fixed it at PR https://github.com/kata-containers/packaging/pull/1114#pullrequestreview-474501018. But I made a mistake to place the kernel patch to a wrong place, So I open another PR(https://github.com/kata-containers/packaging/pull/1121) to correct it. I think this error will go away after this PR merged.

**Comment by likebreath** [comment]
As tested by the `kata-containers-tests-ARM-18.04-PR-firecracker` CI job from this PR https://github.com/kata-containers/tests/pull/2810 (job [170](http://jenkins.katacontainers.io/job/kata-containers-tests-ARM-18.04-PR-firecracker/170/console) passed), this issue now is being fxed by PR #1121.

**Comment by likebreath** [comment]
> So I open another PR(#1121) to correct it. I think this error will go away after this PR merged.

@jongwu Thanks for fixing the issue. One tip to share is that, as we don't have much CI coverage for the packaging repo, what I did was opening a dummy PR from the kata-tests repo and leveraging the `depend-on` feature to test the changes in the packaging repo. 

One example is that I used this dummy PR https://github.com/kata-containers/tests/pull/2733 (from the kata-tests repo) to verify the other PR https://github.com/kata-containers/tests/issues/2546 (from the kata-packaging repo). The trick is the `depend-on` line in the commit message in the dummy PR. I hope it helps.
